src/components/SingleList.jsx

```diff
@@ -1,19 +1,42 @@
 import { useNavigate } from 'react-router-dom';
 
 import './SingleList.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTrashCan } from '@fortawesome/free-regular-svg-icons';
+import { deleteSingleList } from '../api/firebase';
+import { useAuth } from '../api';
 
 export function SingleList({ name, path, setListPath }) {
+	const { user } = useAuth();
+
 	const navigate = useNavigate();
 
 	function handleClick() {
 		setListPath(path);
 		navigate('list');
 	}
-
+	async function deleteSelectedList() {
+		if (
+			window.confirm(
+				`Are you sure you want to delete the ${name} list? This action cannot be undone and all items in this list will be deleted.`,
+			)
```

Comment:
I know you all are using a library for the modals, but it would be great if we could change the "OK" button in this message in particular to "Delete" and make it `red`, since that's the industry standard.

Reply:
Thank you for this recommendation, @polly89! I'll take the time to explore that further.